checkov/terraform/context_parsers/parsers/locals_context_parser.py

```diff
@@ -0,0 +1,29 @@
+from checkov.terraform.context_parsers.base_parser import BaseContextParser
+import dpath.util
+
+
+class LocalsContextParser(BaseContextParser):
+    def __init__(self):
+        definition_type = "locals"
+        super().__init__(definition_type=definition_type)
+
+    def _collect_local_values(self, local_block):
+        for local_name, local_value in local_block.items():
+            local_value = local_value[0]
+            if type(local_value) in (int, float, bool, str):
+                dpath.new(self.context, ['assignments', local_name], local_value)
+
+    def get_block_type(self):
+        return self.definition_type
+
+    def get_entity_context_path(self, entity_block):
```

Comment:
what is this method used for?

Reply:
@schosterbarak given an entity (may be a `resource`, `data`, `provider` etc.), return the entity's path in the context parser object. It is an overridden method in each context parser that implements the base context parser.

Reply:
great. maybe we should add that in a pydoc on that method? :)